CHANGELOG.md

```diff
@@ -64,6 +64,18 @@
   removed. If your code previously relied on `sor.status() = new_status` you
   should change it to `sor = new_status`. ([#7150](https://github.com/googleapis/google-cloud-cpp/pull/7150))
 
+
```

Comment:
FWIW, it looks like we only have single-blank-line separators between all the other releases.